website/source/guides/node-draining.html.md

```diff
@@ -0,0 +1,247 @@
+completes it will remain ineligible to prevent refilling a newly drained node.
+
+However, by default canceling a drain with the `-disable` option will reset a
+node to be eligible for scheduling. To cancel a drain and preserving the node's
+ineligible status use the `-keep-ineligible` option.
+
+Scheduling eligibility can be toggled independently of node drains by using the
+[`nomad node eligibility`][eligibility] command.
+
+### Node Drain Deadline
+
+Sometimes a drain is unable to proceed and complete normally. This could be
+caused by not enough capacity existing in the cluster to replace the drained
+allocations or by replacement allocations failing to start successfully in a
+timely fashion.
+
+Operators may specify a deadline using the option for node drain to prevent
+drains from getting stuck. Once the deadline is reached, all remaining
+allocations on the node are stopped regardless of `migrate` stanza parameters.
+
+The default deadline is 1 hour and may be changed with the
+[`-deadline`][deadline] command line option. The [`-force`][force] option is
+like an instant deadline: all allocations are immediately stopped. The
+[`-no-deadline`][no-deadline] option disables the deadline so a drain may
+continue indefinitely.
+
+Like all other drain parameters, a drain's deadline can be updated by making
+subsequent `nomad node drain ...` calls with updated values.
+
+## Node Drains and Non-Service Jobs
+
+So far we have only seen how draining works with service jobs. Both batch and
+system jobs are have different behaviors during node drains.
+
+### Draining Batch Jobs
+
+Node drains only migrate batch jobs once the drain's deadline has been reached.
+For node drains without a deadline the drain will not complete until all batch
+jobs on the node have completed (or failed).
+
+The goal of this behavior is to avoid losing progress a batch job has made by
+forcing it to exit early.
+
+### Keeping System Jobs Running
+
+Node drains only stop system jobs once all other allocations have exited. This
+way if a node is running a log shipping daemon or metrics collector as a system
+job, it will continue to run as long as there are other services running.
+
+The [`-ignore-system`][ignore-system] option leaves system jobs running even
+after all other allocations have exited. This is useful when system jobs are
+used to monitor Nomad itself or other system properties.
+
+## Draining Multiple Nodes
+
+A common operation is to decommission an entire class of nodes at once. Prior
+to Nomad 0.7 this was a problematic operation as the first node to begin
+draining may migrate all of their allocations to the next node about to be
+drained. In pathological cases this could repeat on each node to be drained and
+cause allocations to be rescheduled repeatedly.
+
+As of Nomad 0.8 an operator can avoid this churn by marking node ineligible for
+scheduling before draining them using the [`nomad node
+eligibility`][eligibility] command:
+
+```text
```

Comment:
I think we can improve this example a bit. I would suggest:

For example, assume we have two nodes in DC1 and we want to migrate all the workload to the new DC2.

Display `node status` output with three nodes in each, all six nodes eligible. (nomad-1 through nomad-6)

To avoid drains from nodes in DC1 migrating to other nodes in DC1, we first mark all nodes in DC1 as ineligible for scheduling. (show output) and then then we mark them each for drain, migrating all workloads to the new DC2.